ros2/source/source/Dockerfile

```diff
@@ -72,17 +27,20 @@ RUN apt-get update && rosdep install -y \
     && rm -rf /var/lib/apt/lists/*
 
 # build source
-WORKDIR $ROS2_WS
 RUN colcon \
     build \
     --cmake-args -DSECURITY=ON --no-warn-unused-cli \
     --symlink-install
 
-# setup bashrc
-RUN cp /etc/skel/.bashrc ~/
-
-# setup entrypoint
-COPY ./ros_entrypoint.sh /
+ARG RUN_TESTS
+ARG FAIL_ON_TEST_FAILURE
+RUN if [ ! -z "$RUN_TESTS" ]; then \
+        colcon test; \
+        if [ ! -z "$FAIL_ON_TEST_FAILURE" ]; then \
+            colcon test-result; \
+        else \
+            colcon test-result || true; \
+        fi \
+    fi
 
-ENTRYPOINT ["/ros_entrypoint.sh"]
 CMD ["bash"]
```

Comment:
Setting of the command in `source` is also redundant given its set in the `devel` image.

Reply:
Good point.
I originally kept the template logic in case we wanted to provide more commands to be set here.
I don't see a strong use case for it though so I'll remove it completely from the template.

Does that sound good?

Reply:
removed in https://github.com/osrf/docker_templates/pull/57/commits/d9e085206f4c66e661cdd314c6675df371fbf6bc resulting in https://github.com/osrf/docker_images/pull/246/commits/b4924e918e81524c4e05570aa91cec1495cdd247